### PR TITLE
Centralized plan info

### PR DESCRIPTION
**Pull request by kodster28**

### CODE REVIEW

**Comment by kodster28** [comment]
@WalshyDev, this'll probably be a slow burn.

**Comment by WalshyDev** [comment]
👀 

Ty Kody!!

**Comment by kodster28** [comment]
Absolutely! And now I think I've got a standardized component for `<feature-table>`, as well, so you don't have to specify so many values.

Not quite sure it fits the Pages/Workers use case yet, but should be testing that out soon-ish.

File: content/ssl/edge-certificates/custom-certificates/_index.md [hunk 1]
```diff
@@ -38,3 +36,7 @@ Each pack only counts as one SSL certificate against your custom certificate quo
 You cannot delete the primary certificate if secondary certificates are present in the pack.
 
 {{</Aside>}}
+
+## Availability
+\
```
Comment (pedrosousa) [inline content/ssl/edge-certificates/custom-certificates/_index.md]: Do we need this backslash? It's adding an extra empty line.

File: data/plans.json [hunk 2]
```diff
@@ -0,0 +1,2393 @@
...
+          "pro": 5,
+          "biz": 5,
+          "ent": 25
+        },
+        "url_redirects": {
+          "title": "URL redirects across lists",
+          "free": 20,
+          "pro": 500,
+          "biz": 500,
+          "ent": 10000
```
Comment (pedrosousa) [inline data/plans.json]: Maybe these values should be strings, otherwise we will lose the thousands separator in "10,000".

File: data/plans.json [hunk 3]
```diff
@@ -0,0 +1,2393 @@
...
+        }
+      }
+    },
+    "custom_certificates": {
+      "title": "Custom Certificates",
+      "link": "/ssl/edge-certificates/custom-certificates/",
+      "properties": {
+        "availability": {
+          "title": "Availability",
+          "summary": "Business and Enterprise customers get one certificate by default and Enterprise customers can purchase additional custom certificates if needed.",
```
Comment (pedrosousa) [inline data/plans.json:1686]: This is one of the few full sentences we have for `summary` (and the only `summary` being used). We should probably decide what's the plan for the `summary` field: either just a couple of words or full sentences.

File: data/plans.json [hunk 4]
```diff
@@ -0,0 +1,2393 @@
...
+        "availability": {
+          "title": "Availability",
+          "summary": "Pro and above",
+          "free": "No",
+          "pro": "Yes",
+          "biz": "Yes",
+          "ent": "Yes",
+          "ent_plus": "Yes"
+        },
+        "b_script_monitor": {
```
Comment (pedrosousa) [inline data/plans.json:1202]: Using `b_`, `c_`, etc. seems a bit of a hack for defining row order... Do Hugo templates always handle JSON entries alphabetically?

File: data/plans.json [hunk 5]
```diff
@@ -0,0 +1,2393 @@
...
+          "pro": "No",
+          "biz": "No",
+          "ent": "Certain Enterprise plans"
+        }
+      }
+    },
+    "page_shield": {
+      "title": "Page Shield",
+      "link": "/page-shield/",
+      "ent_plus": "Enterprise with add-on",
```
Comment (pedrosousa) [inline data/plans.json:1191]: Rate limiting availability would require two `ent_plus` columns, or a different approach:
https://centralized-plan-info.cloudflare-docs-7ou.pages.dev/waf/rate-limiting-rules/#availability

File: content/rules/url-forwarding/_index.md [hunk 6]
```diff
@@ -19,18 +19,14 @@ Use the following Cloudflare products to perform URL redirects, according to you
...
+### Bulk redirects
 
-Single Redirects features and quotas are per zone and depend on the zone plan.
+{{<feature-table id="rules.bulk_redirects">}}
+
+For *URL redirects across lists*, this table provides the default quota for the Enterprise plan. Bulk Redirects supports several million URL redirects — to get more redirects, contact your account team.
+
+### Single Redirects
 
-Bulk Redirects features and quotas are per account and they depend on the highest Cloudflare plan on your account.
```
Comment (pedrosousa) [inline content/rules/url-forwarding/_index.md:36]: We seem to have lost this sentence.

File: data/plans.json [hunk 7]
```diff
@@ -0,0 +1,2393 @@
...
+        "url_redirects": {
+          "title": "URL redirects across lists",
+          "free": 20,
+          "pro": 500,
+          "biz": 500,
+          "ent": 10000
+        }
+      }
+    },
+    "cache_rules": {
```
Comment (pedrosousa) [inline data/plans.json]: I'd expect these rules to be under "Cache", similarly to what happens in the Cloudflare dashboard.

**Review by pedrosousa — APPROVED** [review]

File: data/plans.json (hunk 4 above)
Comment (kodster28) [reply data/plans.json:1202]: Yeah, unfortunately. I think it's worth longer-term investigation, but probably not able to fix by this release.

File: data/plans.json (hunk 3 above)
Comment (kodster28) [reply data/plans.json:1686]: That's definitely a good idea, but I'll probably wait to standardize until after this release.

File: data/plans.json (hunk 5 above)
Comment (kodster28) [reply data/plans.json:1191]: Yeah... probably one of those post-go live standardizations. Probably an open question as to how common that is as a need.

**Pull request merged**